settings.gradle

```diff
@@ -25,6 +25,7 @@ includeWithFlags ':logback',                             'java', 'publish', 'rel
 includeWithFlags ':oauth2',                              'java', 'publish', 'relocate'
 includeWithFlags ':protobuf',                            'java', 'publish', 'relocate'
 includeWithFlags ':reactor3',                            'java', 'publish', 'relocate'
+includeWithFlags ':resilience4j2',                       'java17', 'publish', 'relocate'
```

Comment:
Note: It may be worth supporting Resilience4j 1.x if the APIs are compatible and we can share the code with a different `build.gradle` file. Because it would take a relatively long time to migrate Java 8 or Java 11 to Java 17 for community users and LINE internal users.
But I don't think we need to handle it in this PR. We may add it later in the following PR.

Reply:
Agree with this.

If my memory serves correctly, I only modified some tests and production code stayed the same.
I'll add this in a follow-up PR. 👍

Reply:
Sounds good! Looking forward to it.